src/pages/careers/data/_career-data.ts

```diff
@@ -730,6 +730,69 @@ const career_data = {
             ],
         },
     ],
+    business_intelligence: [
+        {
+            header: {
+                background: 'business_intelligence_bg',
+                title: 'Business Intelligence',
+                subtitle:
+                    'Our team leads the organisation in cultivating a data-driven culture as our company moves towards the future. We collect meaningful data and analytics to drive smart marketing decisions, optimise our business, and increase profitability.',
+            },
+            intro_section: {
+                image: 'business_intelligence_desc',
+                img_alt: 'Project management team on online call',
+                paragraph: [
+                    {
+                        text: 'We employ industry-standard practices when designing and improving data solutions for analytics, warehousing, and more. In this fast-paced environment, you’ll be involved in projects that have an impact on the direction and growth of the company. While you’ll have the opportunity to learn from experts, your ideas for improvement are welcomed as we move towards a data-driven company.',
+                    },
+                ],
+            },
+            link_url: 'https://deriv.zohorecruit.eu/jobs/Business-Intelligence',
+            open_position: [
+                {
+                    title: 'Senior Data Analyst',
+                    location: 'Cyberjaya, Malaysia',
+                    id: '47807000001891570',
+                },
+                {
+                    title: 'Senior Data Engineer',
+                    location: 'Cyberjaya, Malaysia',
+                    id: '47807000001891250',
+                },
+                {
+                    title: 'Senior Data Analyst',
+                    location: 'Limassol, Cyprus',
+                    id: '47807000001891704',
+                },
+                {
+                    title: 'Senior Data Engineer',
+                    location: 'Limassol, Cyprus',
+                    id: '47807000001891732',
+                },
```

Comment:
Let's add 4 items. Rest of them can be visible from the view zoho page

Reply:
Will make the changes @habib-deriv . Thanks